clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -3205,6 +3205,40 @@ void CIRGenModule::applyReplacements() {
     auto NewF = dyn_cast<cir::FuncOp>(Replacement);
     assert(NewF && "not implemented");
 
+    // LLVM has opaque pointer but CIR not. So we may have to handle these
+    // different pointer types when performing replacement.
+    auto optionalUseRange = OldF.getSymbolUses(theModule);
```

Comment:
Please split this into a helper function.

Reply:
Done